libs/features/personalization/entitlements.js

```diff
@@ -19,8 +19,10 @@ const ENTITLEMENT_MAP = {
   'eda8c774-420b-44c2-9006-f9a8d0fb5168': '3d-substance-texturing',
   // PEP segments
   '6cb0d58c-3a65-47e2-b459-c52bb158d5b6': 'lightroom-web-usage',
-  'caa3de84-6336-4fa8-8db2-240fc88106cc': 'photoshop-web-usage',
-  '5c6a4bb8-a2f3-4202-8cca-f5e918b969dc': 'firefly-web-usage',
+  'caa3de84-6336-4fa8-8db2-240fc88106cc': 'photoshop-signup-source',
+  'ef82408e-1bab-4518-b655-a88981515d6b': 'photoshop-web-usage',
+  '5c6a4bb8-a2f3-4202-8cca-f5e918b969dc': 'firefly-signup-source',
+  '20106303-e88c-4b15-93e5-f6a1c3215a12': 'firefly-web-usage',
```

Comment:
Do they all live in cc only? We might want to start using the `consumerEntitlements` config, instead of having that list grow here.

Reply:
I'm not super familiar with how we're handling entitlements in general. I see that currently we should have the same content in both `entitlements` and `consumerEntitlements`. From lines 233-4 of libs/utils/utils.js:
```javascript
      config.entitlements = handleEntitlements;
      config.consumerEntitlements = conf.entitlements || [];
```
I'm not sure how the difference between the two maps on to a distinction in how we think about regular entitlements vs consumer entitlements.

Reply:
I _think_ it's just the default, what you seeing the utils. If a consumer defines them, it should overwrite them & then be considered here: https://github.com/adobecom/milo/blob/6cb99ae2b4847c00ddcf183a7469c523328050fd/libs/features/personalization/entitlements.js#L29

@chrischrischris is that correct?